cachi2/core/models/sbom.py

```diff
@@ -28,6 +28,25 @@ class ExternalReference(pydantic.BaseModel):
     type: Literal["distribution"] = "distribution"
 
 
+class PatchDiff(pydantic.BaseModel):
```

Comment:
The CycloneDX schema offers the option of a text diff for the patch, but it is not required. I opted not to include so as not to clutter the SBOM. Let me know if you disagree.

Reply:
We now have a precedent :) in https://github.com/containerbuildsystem/cachi2/blob/38f8cd9d79669c60edc134d96ef0c69822d371db/cachi2/core/models/input.py#L217, so we could enable this the same way eventually.